bower.json

```diff
@@ -1,10 +1,6 @@
 {
   "name": "ember-cli-simple-validation",
   "dependencies": {
-    "ember": "1.13.13",
-    "ember-cli-shims": "0.1.0",
-    "ember-cli-test-loader": "0.2.2",
-    "ember-qunit-notifications": "0.1.0",
-    "jquery": "~2.1.4"
+    "ember": "1.13.13"
```

Comment:
Is bower.json still required?

Reply:
The addon’s dummy app was built with ArrayController so I’ve avoided the work to update it beyond ember 1.13.13 and last I checked bower is required to install this older version of the framework with ember cli

Reply:
Gotcha. Sounds good.